### PR TITLE
fix(datepicker): improve a11y

### PR DESCRIPTION
**Pull request by nadiia**
Keyboard navigation:
- fix tabbing in the calendar when a display month changed
- fix the calendar focus on the arrow key down from the input when the focus previously changed by a click into the input element

### CODE REVIEW

File: src/datepicker/__tests__/datepicker-unreliable.e2e.js [hunk 1]
```diff
@@ -136,8 +135,7 @@ describe('Datepicker', () => {
     await page.click(selectors.rightArrow);
     const value = await page.$(selectors.monthYearSelectButton);
     const text = await page.evaluate(element => element.textContent, value);
-    // (Month YearTriangle Down) because it renders an icon within the element
-    expect(text).toBe('December 2030Triangle Down');
+    expect(text).toBe('December 2030');
```
Comment (gergelyke) [inline src/datepicker/__tests__/datepicker-unreliable.e2e.js]: 👏

File: src/datepicker/calendar-header.js [hunk 2]
```diff
@@ -405,7 +407,7 @@ export default class CalendarHeader extends React.Component<
         >
           {monthYearTitle}
           <MonthYearSelectIconContainer {...monthYearSelectIconContainerProps}>
-            <TriangleDown />
+            <TriangleDown title="" />
```
Comment (gergelyke) [inline src/datepicker/calendar-header.js]: shouldn't we use `aria-hidden` instead?

Comment (tajo) [reply src/datepicker/calendar-header.js]: and `role="presentation"`

**Comment by tajo** [comment]
works nicely!

File: src/datepicker/calendar-header.js (hunk 2 above)
Comment (nadiia) [reply src/datepicker/calendar-header.js]: yeah, I can add the 'presentation' role but the title still seems not useful here

**Pull request merged**